File: VB/B3013/MainForm.vb
Imports DevExpress.XtraEditors

Namespace B3013
    Partial Public Class MainForm
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Protected Overrides Sub OnShown(ByVal e As System.EventArgs)
            MyBase.OnShown(e)
            myTreeList1.OptionsView.RowImagesShowMode = DevExpress.XtraTreeList.RowImagesShowMode.InCell
        End Sub
    End Class
End Namespace

File: CS/B3013/MainForm.cs
using DevExpress.XtraEditors;

namespace B3013 {
    public partial class MainForm :XtraForm {
        public MainForm() {
            InitializeComponent();
        }
        protected override void OnShown(System.EventArgs e) {
            base.OnShown(e);
            myTreeList1.OptionsView.RowImagesShowMode = DevExpress.XtraTreeList.RowImagesShowMode.InCell;
        }
    }
}